### ISSUE TITLE
Rendering of blank page after usual ajax head response

### ISSUE THREAD

**Issue by omitter** [jorgemanrubia/turbolinks_render#12]
When I make a usual ajax call where I'm only interested in the status code I usually use `head` instead of `render`ing anything. It looks like turbolinks_render falsely intercepts these calls and substitutes the whole page with a blank string.

example script:
```coffeescript
    $.ajax
      url: "/posts/" + post_id + "/like",
      method: 'PUT',
      success: () ->
        $this.toggleClass("clicked")
```
example controller:
```ruby
  def like
    if Post.find(params[:post_id]).like!(@user)
      head :ok
    else
      head :forbidden
    end
  end
```

**Comment by jorgemanrubia** [comment]
Thanks for the report @omitter. I'll see how to fix this.

**Comment by jorgemanrubia** [comment]
Hey @omitter I couldn't reproduce with a quick test. Are you using the last version of the gem `0.9.11`? I still need to test more, but I was wondering if you might be using an older version here.

**Comment by jorgemanrubia** [comment]
Nevermind I could reproduce, I'll see how to fix soonish

**Comment by jorgemanrubia** [comment]
I just released version `0.9.12` fixing this. Thanks for the bug report @omitter!

**Comment by omitter** [comment]
Great, I can confirm that it works. Thanks!
